### DISCUSSION TITLE
Containers do not use Tailwind's container configuration

### DISCUSSION THREAD

**Discussion by Kturva** [livewire/flux#129] [Feature Requests]
Currently containers from `flux:container`, `flux:header` and `flux:main` are hard coded to `mx-auto w-full max-w-7xl`. It might be better to default to user's Tailwind's configuration (https://tailwindcss.com/docs/container) to avoid having to publish and edit the stubs.

**Comment by dudelisius** [comment]
I think this is a very opinionated thing. Personally I prefer the way flux does it (I've done it with these classes myself) and feel these work better in applications than tailwinds container.

**Comment by joshhanley** [comment]
@Kturva thanks for reporting! I have converted this issue to a feature request for now, so we can focus on fixing any bugs first.
